net-irc/hexchat/hexchat-2.12.4.ebuild

```diff
@@ -60,8 +60,10 @@ PATCHES=(
 )
 
 src_prepare() {
+	if [[ ${PV} == "9999" ]]; then
+		eautoreconf
+	fi
 	default
-	eautoreconf
```

Comment:
don't change the order! the current call order means you can never patch the build system in a way that it propagates down. Always call eautoreconf after applying patches.